src/plugins/compressors/sz_plugin.cc

```diff
@@ -49,15 +49,13 @@ namespace {
     {"abs_and_pw_rel", ABS_AND_PW_REL},
     {"rel_or_pw_rel", REL_OR_PW_REL},
     {"rel_and_pw_rel", REL_AND_PW_REL},
-  };
-}
+};
 
 class sz_plugin: public libpressio_compressor_plugin {
   public:
   sz_plugin() {
     std::stringstream ss;
-    ss << sz_plugin::major_version() << "." << sz_plugin::minor_version() << "." << sz_plugin::patch_version() << "." << revision_version();
-    sz_version = ss.str();
+    sz_version = get_version_sz(sz_plugin::major_version(),sz_plugin::minor_version(),sz_plugin::patch_version(),revision_version());
```

Comment:
This isn't the way to do this.  We can talk about this, or I can fix it and explain to you want I did.